src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -86,7 +86,6 @@ class AuthScreens extends React.Component {
         super(props);
 
         Timing.start(CONST.TIMING.HOMEPAGE_INITIAL_RENDER);
-        Timing.start(CONST.TIMING.HOMEPAGE_REPORTS_LOADED);
```

Comment:
This timer was being started but never stopped / recorded, because the stop for it was in `fetchAllReports`, which was no longer being called anywhere.